src/arkit.cs

```diff
@@ -505,6 +507,7 @@ interface ARWorldTrackingConfiguration {
 	[iOS (11,0)]
 	[NoWatch, NoTV, NoMac]
 	[BaseType (typeof(ARConfiguration))]
+	[DisableDefaultCtor]
```

Comment:
Sorry if it was confusing ;-) 
_but in this case it might be valid but..._
meant that I did not knew if `init` made sense - because the base type did not have it but *was* abstract
It turns out it's valid
https://developer.apple.com/documentation/arkit/arorientationtrackingconfiguration/2923556-init?language=objc

Reply:
Ah sorry I misinterpreted that and thought you checked (: